go/integTest/readfrom_strategy_test.go

```diff
@@ -22,16 +23,19 @@ func (suite *GlideTestSuite) TestRoutingWithAzAffinityStrategyTo1Replica() {
 	clientForConfigSet := suite.clusterClient(suite.defaultClusterClientConfig().WithRequestTimeout(2000))
 
 	// Reset the availability zone for all nodes
-	_, err := clientForConfigSet.CustomCommandWithRoute(
-		[]string{"CONFIG", "SET", "availability-zone", ""}, config.AllNodes)
+	_, err := clientForConfigSet.ConfigSetWithOptions(context.TODO(),
```

Comment:
Good catch on CustomCommand. We should probably do a search to make sure there aren't other CustomCommand usages that are in code at this point.